### ISSUE TITLE
Skia-rendered content behaving like rubber

### ISSUE THREAD

**Issue by Enyium** [slint-ui/slint#5765]
I'm on Windows 10 using Rust.

I only have this with Skia, not with FemtoVG or the software renderer (Winit backend): When I resize the window, the content is somehow resized with the window (obviously by the GPU like in a video player). Then it goes back to the correct size, but strangely in a smooth way.

Video recording of the effect:

https://github.com/user-attachments/assets/1263293f-9862-4f77-9dce-ce20644270c1

Obviously, noone would want that.

**Comment by tronical** [comment]
That sounds indeed like a bug. Does it work better for you if you select `renderer-skia-opengl` instead of `renderer-skia`?

**Comment by Enyium** [comment]
Indeed, with the `renderer-skia-opengl` Cargo feature, it works correctly.

**Comment by Horbin-Magician** [comment]
I also meet the same problem even using `renderer-skia-opengl`. This problem may look different on different GPUs (Works well in Intel and Nvidia, but not in AMD).

**Comment by the-argus** [comment]
@tronical this was resolved by 0d7904e62 :D

On windows with AMD graphics I am able to reproduce it before then with the d3d surface, and not afterwards
